web/tests/test_chapters.py

```
@@ -0,0 +1,634 @@
+    Chapter,
+    ChapterApplication,
+    ChapterEvent,
+    ChapterEventAttendee,
+    ChapterMembership,
+    ChapterResource,
+)
+
+
+class ChapterModelTests(TestCase):
+    def setUp(self) -> None:
+        self.user = User.objects.create_user(username="testuser", email="test@example.com", password="password123")
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "password"

(S106)